scripts/fetch/fetch.states.R

```diff
@@ -29,7 +29,15 @@ fetch_map_data <- function(..., viz){
 }
 
 fetch.states <- function(viz = as.viz('states')){
-  fetch_map_data('state', viz = viz)
+  
+  view <- readDepends(viz)[['view-limits']]
+  states.out <- to_sp('state', proj.string = view[['proj.string']],
+                      within = as.sp_box(view$xlim, view$ylim, CRS(view[['proj.string']])))
+  mexico.out <- to_sp('world','Mexico', proj.string = view[['proj.string']],
+                      within = as.sp_box(view$xlim, view$ylim, CRS(view[['proj.string']])))
+  
+  out <- rbind(states.out, mexico.out)
+  saveRDS(out, viz[['location']])
```

Comment:
Nice. I was so confused by the state labels from Irma. Is this all it took to fix it?